src/runtime/lpython_parser.py

```diff
@@ -96,7 +96,7 @@ def generic_visit(self, node):
                 raise Exception("Unsupported value type")
         new_ast = getattr(python_ast, class_name)
         new_node = new_ast(**d)
-        if hasattr(node, "lineno"):
+        if hasattr(node, "col_offset"):
```

Comment:
This change was made to fix the Old Parser issue: https://github.com/lcompilers/lpython/issues/788#issue-1309054967